llvm/test/CodeGen/Hexagon/hvx-vdeal-vpack.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc -march=hexagon -O2  < %s | FileCheck %s
```

Comment:
IIRC in general, `-mtriple` is preferable over `-march`.

For this test case it might not matter, though.